@types/phaser3-rex-plugins/plugins/behaviors/textedit/TextEdit.d.ts

```diff
@@ -0,0 +1,17 @@
+declare module 'phaser3-rex-plugins/plugins/behaviors/textedit/TextEdit' {
+  export default class TextEdit {
+    constructor(gameObject: Phaser.GameObjects.GameObject);
+    gameObject: Phaser.GameObjects.GameObject
+    scene: Phaser.Scene
+    inputText: string
+    onClose: Function
+    delayCall: Phaser.Time.TimerEvent
+    boot(): this
+    shutdown(): this
+    destroy(): this
+    open(config: any, onCloseCallback: Function): this
```

Comment:
- config should be interface with 2 optional `onClose?` and `onTextChanged?` params
- Looks like it should be 2 function overloads:
`open(config: any, onCloseCallback: Function): this`
and
`open(onCloseCallback: Function): this`
- onCloseCallback could be more specific callback that take GameObject as first param.